cypress/support/commands.ts

```diff
@@ -77,6 +77,30 @@ export const createProjectWithFixture = (
   });
 };
 
+export const createEmptyProject = (
+  name: string = "new-project",
+  description: string,
+  path: string,
+  defaultBranch: string = "trunk",
+  localhost: number = 17246
+): Cypress.Chainable<void> =>
```

Comment:
It’s probably also a good idea to only have parameters that actually make a difference in the test code. We can remove `defaultBranch` and `description` and just hard-code it.